### PR TITLE
added limitation docs

### PR DESCRIPTION
**Pull request by aksh-h**

### CODE REVIEW

File: docs/Limitations.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Limitations of Azure DevOps Demo Generator
```
Comment (dmckinstry) [inline docs/Limitations.md:1]: I'll have some minor verbiage changes, but let's get this PR through and then I'll PR back to you with those changes...

File: docs/Limitations.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Azure DevOps Demo Generator is to create a demo set up with predefined templates, it has some limitations, which are listed below. This can't be treated as migration or copying project tool from one Organization to another.
+
+## Following are the limitations of Azure DevOps Demo Generator and Template Builder
+
+|||
+|------|-----|
+|Custom process template|This is supported to an extent, but not all the options available as compared to standard process template. Organization must contains same process template Id|
```
Comment (dmckinstry) [inline docs/Limitations.md:9]: I want to make sure I understand...  this is "Fully supported except that the exact same custom template must already exist in the target organization."  Is that correct?  Are there still limitations even if that is true? And if that isn't true, does it still provide a partial migration or is it a complete fail?

File: docs/Limitations.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+| Iterations| Not all the iterations can be exported by the Template Builder with hierarchy|
```
Comment (dmckinstry) [inline docs/Limitations.md:10]: What does "not all" mean?  Are we saying that only a specific subset are exported?  Perhaps a hierarchy depth?  Or only the built-in iterations?

File: docs/Limitations.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+|Iteration dates| It has predefined logic to align the dates, this can't be changed based on the requirement |
+| Work Items | Not all the work items will be exported. This count is limited to 200 work items of each type, if there are more than 200 work items |
+|Work item fields|Only few standard fields are supported, ex: Title, state, iteration path, Description, Area Path |
```
Comment (dmckinstry) [inline docs/Limitations.md:13]: This is good info...  Is this a fixed list or is there other guidance that we can give to make it more predictable to users?

File: docs/Limitations.md (hunk 2 above)
Comment (aksh-h) [reply docs/Limitations.md:9]: Though having same process template there will be some limitations with setting up board colums. Rest other things like work items, pipelines, repose will work as is.

**Pull request merged**